pkgs/by-name/fa/fastfetch/package.nix

```diff
@@ -118,6 +120,8 @@ stdenv'.mkDerivation (finalAttrs: {
 
   cmakeFlags = [
     (lib.cmakeOptionType "filepath" "CMAKE_INSTALL_SYSCONFDIR" "${placeholder "out"}/etc")
+    (lib.cmakeOptionType "filepath" "CUSTOM_PCI_IDS_PATH" "${hwdata}/share/hwdata/pci.ids")
```

Comment:
looks like we're only including hwdata in build inputs for linux atm, can move that to the linux section with libdrm too

Reply:
Many thanks for the review. Have amended with the requested changes. I think that is correct now.